pytorch/requirements.txt

```diff
@@ -3,7 +3,7 @@ plotly
 pytorch-ignite
 pytorch-lightning
 skorch
-torch
-torchvision
-torchaudio
+torch==2.0.0
+torchvision==0.15.1
+torchaudio==2.0.1
```

Comment:
How about leaving these without version specification? I think that it should work.

Reply:
Sounds reasonable and it might enable the tests run in python 3.7.